service/service.go

```diff
@@ -0,0 +1,64 @@
+package service
+
+import (
+	"encoding/base64"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/medibloc/panacea-doracle/config"
+	"github.com/medibloc/panacea-doracle/crypto"
+	"github.com/medibloc/panacea-doracle/panacea"
+	"github.com/medibloc/panacea-doracle/sgx"
+	"github.com/medibloc/panacea-doracle/types"
+	"os"
+	"path/filepath"
+)
+
+type Service struct {
+	Conf          *config.Config
+	OracleAccount *panacea.OracleAccount
+	OraclePrivKey *btcec.PrivateKey
+	UniqueID      string
+
+	QueryClient *panacea.QueryClient
+	GrpcClient  *panacea.GrpcClient
+}
+
+func New(conf *config.Config) (*Service, error) {
+	homeDir, err := os.UserHomeDir()
```

Comment:
When I tested, the homeDir set here and [another homeDir](https://github.com/medibloc/panacea-doracle/blob/main/cmd/doracled/cmd/root.go#L36) are different.

```
# gen-oracle-key command
/data/.panacea/oracle_priv_key.sealed

# run command
/data/oracle_priv_key.sealed
```

Reply:
I'll check and fix it. Thank you gyuguen.

Reply:
Good.

It would be good to commonize the part that gets thest paths later.

Reply:
Agreed. I'll do that in another PR.

Reply:
I'm making a function that can set homeDir (`doracle start --home /data/.doracle2`), so let's do this together. :)